main.go

```diff
@@ -387,20 +399,47 @@ func main() {
 		cancel()
 	}()
 
-	sqlDB, err := models.OpenDB(*postgresDBNameF, *postgresDBUsernameF, *postgresDBPasswordF, l.Debugf)
+	sqlDB, err := models.OpenDB(*postgresDBNameF, *postgresDBUsernameF, *postgresDBPasswordF)
 	if err != nil {
 		l.Panicf("Failed to connect to database: %+v", err)
 	}
 	defer sqlDB.Close()
 	db := reform.NewDB(sqlDB, postgresql.Dialect, nil)
 
+	go func() {
```

Comment:
Are you sure that migration should be done in goroutine?

Reply:
Yeah, that's the point.